content/en/cloudprem/processing.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Processing Configuration
+description: Learn how to configure your processing pipelines in CloudPrem
+further_reading:
+- link: "/logs/log_configuration/processors/"
+  tag: "Documentation"
+  text: "Datadog Log Management Processors"
+- link: "/cloudprem/"
+  tag: "Documentation"
+  text: "Learn more about CloudPrem"
+- link: "/cloudprem/installation/"
+  tag: "Documentation"
+  text: "CloudPrem Installation"
+- link: "/cloudprem/ingress/"
+  tag: "Documentation"
+  text: "Ingress Configuration"
+- link: "/cloudprem/cluster/"
+  tag: "Documentation"
+  text: "Cluster Sizing and Operations"
+- link: "/cloudprem/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting CloudPrem"
+---
+
+## Overview
+
+CloudPrem includes a processing feature that allows you to parse and enrich logs. It automatically parses logs formatted in JSON. You can define pipelines and processors to extract meaningful information or attributes from semi-structured text, which can then be used for aggregations. This processing is designed to be equivalent to [Datadog Log Management Pipelines][1]. For a list of supported and unsupported processors, see [Compatibility with Datadog Pipelines](#compatibility-with-datadog-pipelines).
+
+You configure log processing pipelines using a JSON file that adheres to the same format as Datadog pipeline configurations.
+
+## Setting up processing
+
+1. (Optional) If you have existing pipelines in Datadog, you can fetch the configuration using the [Logs Pipelines API][2]:
+
+   ```bash
+   curl -X GET "https://api.datadoghq.com/api/v1/logs/config/pipelines" \
+    -H "Accept: application/json" \
+    -H "DD-API-KEY: ${DD_API_KEY}" \
+    -H "DD-APPLICATION-KEY: ${DD_APP_KEY}" > my-pipelines_config.json
+   ```
+
+   This JSON file can be used directly with CloudPrem.
+
+2. To set the configuration in the Helm Chart, provide the path to your JSON configuration file using the `pipelinesConfig` parameter in the CloudPrem Helm chart:
+
+   ```bash
+   helm repo update
+   helm upgrade <release-name> -n <namespace> --set-file pipelinesConfig=./pipelines_config.json -f <custom_values_file.yaml>
+   ```
+
+   CloudPrem logs an informational message (`Successfully read pipeline config file`) when it successfully reads the configuration file. Any processors defined in the file that are not supported by CloudPrem are ignored during startup.
+   **Note**: Helm imposes a 1 MB size limit on the configuration file due to its underlying etcd storage.
+
+## Configuration file format
+
+The configuration is a JSON array, where each element represents a processor or a nested pipeline.  
+
+The order of elements in the array defines the sequential execution order of the processors.  The structure mirrors the output of the Datadog API endpoint `api/v1/logs/config/pipelines`.
+
+
+```json
+[
+ {...processor1},
+ {...processor2}
+]
+```
+
+```json
```

Comment:
For these json codeblocks, typically each indentation level would be 2 spaces (or 4, but usually 2). Not sure how much it matters to you, it just looks a bit odd if you're used to working with JSON. You can probably just run it through a JSON formatter if you want to fix it quickly.